packages/dnslink-api/src/index.js

```diff
@@ -0,0 +1,78 @@
+const dns = require("dns");
+const express = require("express");
+const NodeCache = require("node-cache");
+const isValidDomain = require("is-valid-domain");
+
+const host = process.env.HOSTNAME || "0.0.0.0";
+const port = Number(process.env.PORT) || 3100;
+
+const server = express();
+const cache = new NodeCache({ stdTTL: 300 }); // cache for 5 minutes
+
+const dnslinkRegExp = /^dnslink=.+$/;
+const skylinkDnslinkRegExp = /^dnslink=\/skynet-ns\/([a-zA-Z0-9_-]{46}|[a-z0-9]{55})/;
```

Comment:
We should define a skylink-matching regex somewhere and reuse it. We have a bunch of those defined across the project, most only matching 46-char skylinks.

Reply:
yeah but concatenating regexes is not possible, we could however have it as a string and then initialise with `new RegExp(regexString)` - I guess we could put that in skynet-js or sth